### ISSUE TITLE
US-003

### ISSUE THREAD

**Issue by mattgarrish** [daisy/epub-accessibility-iso#27]
This document is about accessibility. It would be in our best interest to provide   it in an accessible format. EPUB or HTML are recommended.

Proposed Change: Please issue the document in accessible format.

**Comment by mattgarrish** [comment]
While I agree with this comment, it's not something to fix with the draft.

**Comment by mattgarrish** [comment]
partially accepted

Actions:
-	Next draft of EPUB Accessibility should be in accessible format.
-	JTC 1 should send a message to ISO recommending that all documents of ISO should be accessible, referring to UN convention for PWD and Marrakesh treaty.
